tests/assets/non-google-fonts.css

```
@@ -0,0 +1,58 @@
+ * @font-face statements, but also plenty of regular font-family declarations, which are no
+ * font families at all.
+ */
+@font-face {
+    font-family: 'Nunito Sans 12pt ExtraLight 12pt';
+    font-style: normal;
+    font-weight: 200;
+    src: url(https://example.com/fonts/nunito-sans-12pt-extralight.woff2) format('woff2');
+}
+
+@font-face {
+    font-family: 'Mulish';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Mulish" (font-family-name-quotes)

(font-family-name-quotes)